date format errors in elasticsearch

I wanted to set this up using elasticsearch 7.6.1 following your readme.

Indexing worked fine whenever I want to search via imap, I get no results and elasticsearch puts a lot of java exceptions amounting to 

> Caused by: java.lang.IllegalArgumentException: failed to parse date field [Mon, 20 Apr 2020 15:00:07 -0400] with format [EEE, d LLL yyyy HH:mm:ss Z||yyyy-MM-dd HH:mm:ss||yyyy-MM-dd]

in the log.

Any idea what might be going wrong, or how I'd fix that? 

(The format looks correct to me?)

--- a/elastic7-schema.json
+++ b/elastic7-schema.json
@@ -81,7 +81,7 @@
         "message-id":{"type": "keyword"},
         "date": {
           "type": "date",
-          "format": "EEE, d LLL yyyy HH:mm:ss Z||yyyy-MM-dd HH:mm:ss||yyyy-MM-dd"
+          "format": "EEE, d MMM yyyy HH:mm:ss Z||yyyy-MM-dd HH:mm:ss||yyyy-MM-dd"
         },
         "from": {
           "type": "text", "analyzer": "email",

